name-badges/makebadges.py

```diff
@@ -0,0 +1,35 @@
+#!/bin/pyhton3
+
+import sys
+import re
+import lxml.etree as ET
```

Comment:
why not just `from lxml import etree`?

Reply:
because that was in the other svg munging scripts we have that i edited this from